### PR TITLE
Fix localized string for Slayer's Presence

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/data/feats.db/slayers-presence.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
                     }
                 ],
                 "selector": "skill-check",
-                "text": "@Localize[PF2E.SpecificRule.UndeadSlayer.SlayersPresence.Note",
+                "text": "@Localize[PF2E.SpecificRule.UndeadSlayer.SlayersPresence.Note]",
```
Comment (TikaelSol) [inline packs/data/feats.db/slayers-presence.json:45]:
```suggestion
                "text": "PF2E.SpecificRule.UndeadSlayer.SlayersPresence.Note",
```
We can skip Localize calls entirely if it's the only text in a note RE

File: static/lang/re-en.json [hunk 2]
```diff
@@ -2161,7 +2161,7 @@
                 "Staff": "Use Staff Two-Handed"
             },
             "UndeadSlayer": {
-                "SlayersBlessing": {
```
Comment (TikaelSol) [inline static/lang/re-en.json:2164]: Did anything use this?

Comment (simonward) [reply static/lang/re-en.json:2164]: It appears to be unused.

File: packs/data/feats.db/slayers-presence.json (hunk 1 above)
Comment (simonward) [reply packs/data/feats.db/slayers-presence.json]: Done.

**Pull request merged**